pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -430,3 +479,19 @@ func validatePatchableSpec(ingressController operatorv1.IngressController, desir
 
 	return true, ""
 }
+
+// Given an IngressControllerList, returns only IngressControllers with the Owner annotation
+func getIngressWithOwnerAnnotation(ingressList operatorv1.IngressControllerList) *operatorv1.IngressControllerList {
```

Comment:
Are the IngressControllers with the Owner annotation and the IngressControllers with the Owner annotation set to CIO the same set? I want to assume they aren't, in which case I think this function may be better named something like `getIngressWithCIOOwnerAnnotation` - what are your thoughts?

Reply:
+1 to making it more specific. I don't think cluster ingres operator adds the annotations but in theory anyone could